libsolidity/CompilerUtils.h

```diff
@@ -30,6 +30,9 @@ namespace solidity {
 
 class Type; // forward
 
+/// The size in bytes of the function (hash) identifier
+static const unsigned int g_functionIdentifierOffset = 4;
```

Comment:
It's not about readability, but about unnecessarily cluttering namespaces. As far as I know, static const POD is perfectly fine in C++11.